SVGKitLibrary/Demo-OSX/SKSVGObject.h

```diff
@@ -0,0 +1,40 @@
+//
+//  SKSVGObject.h
```

Comment:
The name of this looks odd. Is it  trying to do a two letter naming prefix? Should it be SVGK instead?

(since it's in the demo I guess it doesn't hugely matter, but if it's meant to be following the naming scheme, should probably follow the same one as the rest of the library)

Reply:
Actually all of these demo code is from @MaddTheSane 's work. See his demo here: [MaddTheSane-SKSVGObject.h](https://github.com/MaddTheSane/SVGKit/blob/master/XCodeProjectData/Demo-OSX/SKSVGObject.h)

I can rename it to `SVGKObject`, but I think it's just a model to represent the SVG test to make it work for tableView. Is that really necessary ?

Reply:
All class in Objective-C should use prefix. Beucase Objective-C is strict super set of C. Which does not have namespace like C++/Java/Swift or any modern language. If someone name a class, anyone else can not name it the same which will cause naming conflict.

So modern Objective-C tool or demo, always use prefix, even for demo project. I think @MaddTheSane create the OS X demo with `SK` prefix, so this is why his model called `SKSVGObject`...

Reply:
@adamgit So you mean we need to change the prefix of demo ?

Currentlly they use `SKSVG` for prefix, isn't it enough ? The core framework in SVGKit does not use it at all.

Anyway, if you can provide another prefix, I just do a renaming. My aim is to merge the macOS support in the upstream so that I don't need to let our user from SDWebImage to use a fork branch instead.

Reply:
My preference is for ALL demo classes to have NO prefix. They are NOT part of the library,  therefore there is no reason for them to have a prefix!

(but also: SVGK*ImageView classes are clearly part of the library by now - they should be moved out of the Demo folder and into some other folder - maybe the folder for UIKit-specific additions?)